dev/com.ibm.websphere.appserver.thirdparty.eclipselink.3.0/bnd.bnd

```diff
@@ -0,0 +1,129 @@
+#*******************************************************************************
+# Copyright (c) 2020 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+jakartaeeMe: true
+
+Bundle-SymbolicName: com.ibm.websphere.appserver.thirdparty.eclipselink.3.0
+Bundle-Description: EclipseLink JPA
+Bundle-Distributor: IBM
+Bundle-Activator: com.ibm.ws.eclipselink.osgi.EclipseLinkPersistenceActivator
+Bundle-Name: EclipseLink JPA
+Premain-Class: org.eclipse.persistence.internal.jpa.deployment.JavaSECMPInitializerAgent
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-Vendor: Eclipse.org - EclipseLink Project
+Bundle-ManifestVersion: 2
+Specification-Version: 3.0.0
+
+Export-Package: \
+            org.eclipse.persistence.javax.persistence.osgi;version="3.0.0", \
+            org.eclipse.persistence.jpa.config;version="3.0.0", \
+            org.eclipse.persistence.sessions.serializers.kryo;version="3.0.0", \
+            org.eclipse.persistence.internal.jpa.config.*;version="3.0.0", \
+            org.eclipse.persistence.internal.jpa.metadata.graphs;version="3.0.0", \
+            org.eclipse.persistence.internal.jpa.metadata.sop;version="3.0.0", \
+            org.eclipse.persistence.internal.libraries.asm*;version="3.0.0", \
+            org.eclipse.persistence.internal.libraries.antlr.runtime*;version="3.0.0", \
+            org.eclipse.persistence.jpa.jpql*;version="3.0.0", \
+            org.eclipse.persistence.*;version="3.0.0"
+
+Import-Package: \
+            com.ibm.tx.jta;resolution:=optional, \
+            com.ibm.websphere.ras, \
+            com.ibm.ws.Transaction;resolution:=optional, \
+            com.ibm.ws.jpa.container.eclipselink.logging;resolution:=optional, \
+            !javax.activation, \
+            javax.crypto;resolution:=optional, \
+            javax.crypto.spec;resolution:=optional, \
+            javax.enterprise.context.spi;resolution:=optional, \
+            javax.enterprise.inject.spi;resolution:=optional, \
+            javax.imageio;resolution:=optional, \
+            javax.imageio.stream;resolution:=optional, \
+            javax.jms;resolution:=optional, \
+            javax.json;resolution:=optional, \
+            javax.json.stream;resolution:=optional, \
+            javax.management;resolution:=optional, \
+            javax.management.openmbean;resolution:=optional, \
+            javax.naming;resolution:=optional, \
+            javax.persistence;jpa="3.0";consumer="persistenceService";resolution:=optional, \
```

Comment:
what is the purpose of these imports? Should they be removed or just changed to jakarta namespace?

Reply:
This bnd.bnd generates the manifest for the eclipselink bundle -- the one that combines all 5 or so separate eclipselink bundles from artifactory/maven central.  It's still javax.persistence here, as `jakartaeeMe: true` transforms it (remember this is not an eclipselink that is native jakartaee 9 yet).  This will be changed once that transform is no longer necessary.